lib/rooms.coffee

```diff
@@ -22,3 +23,9 @@ Meteor.methods
     data = checkRoom room
     Rooms.update room,
       $set: grid: not data.grid
+
+  roomTouchToggle: (room) ->
+    check room, String
+    data = checkRoom room
+    Rooms.update room,
+      $set: touch: not data.touch
```

Comment:
I think touch enable/disable should be a local variable, not global to the room. The way you have it set up, toggling the button will change it for everyone in the same room!

Reply:
Should be fixed with new commit? Thanks for the feedback!